tests/runner.py

```diff
@@ -1354,30 +1354,40 @@ class BrowserCore(RunnerCore):
   def __init__(self, *args, **kwargs):
     super(BrowserCore, self).__init__(*args, **kwargs)
 
+  def get_browser_open_func():
+    if not EMTEST_BROWSER:
+      print("Using default system browser")
+      return webbrowser.open_new
+    browser_args = shlex.split(EMTEST_BROWSER)
+    # If the given browser is just a type, use the built-in functionality
+    # of the webbrowser module to open that type of browser
```

Comment:
some examples of valid values of what `type` means here might help. or maybe a link to the python docs?

Reply:
Clarified